assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -6,7 +6,10 @@
   style="margin-bottom: 10px;">
   <div ng-transclude flex><!-- expect start time, end time, status details --></div>
   <div row class="text-right text-xs-left">
-    <a href="" ng-click="goChromeless(options)" style="margin-right: 10px;">
+    <a ng-if="state !== 'empty'"
+       href=""
```

Comment:
Should this be `href="javascript:;"` ? per http://stackoverflow.com/questions/5637969/is-an-empty-href-valid